articles/055_ros_parameter_design.md

```diff
@@ -45,26 +45,26 @@ It is useful to consider an ideal system to understand how it would work and how
 An ideal system would support for the combined use cases of ROS 1.0's built-in parameters as well as ROS 1.0's dynamic parameters system.
 Based on that criteria an ideal system would be able to:
 
-- **Set parameter values**
```

Comment:
Looking at the document, it's probably ok to remove the emphasis here and on all the bullet points because they're already offset from the text below them.